tools/files/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
```

Comment:
Curious why this file was included? It isn't part of the CI structure outlined [here](https://github.com/ansible-collections/cloud-content-handbook/tree/main/CI#what-checks-are-run).

Reply:
The [ansible-creator](https://ansible.readthedocs.io/projects/creator/) follows a similar pattern for scaffolding GitHub workflows. I prefer maintaining this consistency to minimize potential rework in the future, when we start using ansible-creator, especially when it becomes capable of scaffolding GitHub workflows separately.